common-dup/src/main/java/org/duracloud/mill/dup/DuplicationPolicy.java

```diff
@@ -35,6 +37,20 @@
     private Map<String, LinkedHashSet<DuplicationStorePolicy>>
         spaceDuplicationStorePolicies = new HashMap<>();
 
+    private LinkedHashSet<DuplicationStorePolicy> defaultPolicies = new LinkedHashSet<>();
+    
+
+    private List<String> spacesToIgnore = new LinkedList<>();
+
+    public LinkedHashSet<DuplicationStorePolicy> getDefaultPolicies() {
```

Comment:
Is there a need for this to return a LinkedHashSet vs just a Set?

Reply:
I believe the issue here was ensuring a consistent ordering of the policies is preserved when persisted.